docs/models.yml

```diff
@@ -300,12 +304,9 @@ committee:
   default_meeting_id:
     type: relation
     to: meeting/default_meeting_for_committee_id
-  member_ids:
-    type: relation-list
-    to: user/committee_as_member_ids
-  manager_ids:
+  user_ids:
     type: relation-list
-    to: user/committee_as_manager_ids
+    to: user/committee_ids
```

Comment:
Does this mean, every user in a committee can also manage the committee?

Reply:
No, that is what the new `committee_management_level` is for.

Reply:
Okay, that sounds good.